app/models/repository.rb

```diff
@@ -350,7 +349,11 @@ def self.check_status(host_type, repo_full_name)
           project.update_attribute(:status, "Removed")
         end
       end
+
+      true
     end
+
+    false
```

Comment:
it didn't seem obvious at first what the return values should be for this method. Should we document it here, e.g. "Returns true if status was updated, and false otherwise."?

Reply:
Any reason to not have a spec to enforce the behavior?

Reply:
I changed around the cases, so that false is the unreachable case and true is a non 404 case. And documented the method.